src/network/webRTC/SubscriptionsManager.ts

```diff
@@ -44,33 +44,31 @@ class SubscriptionsManager {
 		subscriptionToAdd: Subscription[],
 		subscriptionToRemove: Subscription[]
 	): void {
-		if (size(subscriptionToAdd) > 0 || size(subscriptionToRemove) > 0) {
```

Comment:
Why removing this check? If I remember well, the API fails if the two arrays are void